_posts/2016-06-02-app-hooks.md

```diff
@@ -304,8 +308,9 @@ _This event is currently sent only to `project` and `eshop` apps._
 </td></tr></table>
 
 ### `process.apply.start`
-Event hook called when is about to start import process of a project. The response of your app is ignored.
-_This event is currently sent only to `project` and `eshop` apps._
+Event hook called when is about to start import process of a project.
```

Comment:
Well, koukám, že i předtím jsme to měli špatně 😄 Asi moje chyba 🤐... Nicméně nechceš to tu ještě upravit, prosím? :)

`... start apply process of a project.`